src/Ng/Jobs/NgJob.cs

```diff
@@ -17,7 +17,7 @@ public abstract class NgJob
         protected ILoggerFactory LoggerFactory;
         protected ILogger Logger;
 
-        protected int MaxDegreeOfParallelism { get; }
+        protected int MaxDegreeOfParallelism { get; set; } = 8;
```

Comment:
>MaxDegreeOfParallelism [](start = 22, length = 22)

This seems like a risky change. Does this affect feed2catalog/catalog2registration/catalog2lucene?

Please update catalog2packagefixup. See: [https://github.com/NuGet/NuGet.Services.Metadata/blob/master/src/Ng/Jobs/Catalog2PackageFixupJob.cs#L144](https://github.com/NuGet/NuGet.Services.Metadata/blob/master/src/Ng/Jobs/Catalog2PackageFixupJob.cs#L144)

Reply:
Good catch.  This was unintentional.